core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -919,24 +912,20 @@ public function loadDetails()
 						$this->clientCache = $this->clientCache !== false ? $this->clientCache : $objParentPage->clientCache;
 					}
 
+					// Protection
```

Comment:
Why did you move this code block?

Reply:
Guess that must have been an issue during rebase. Fixed in 92007bc2231ed6fe6621812ce1130c394b347895.